src/pages/home/report/ReportActionItemMessageEdit.js

```diff
@@ -101,6 +101,15 @@ class ReportActionItemMessageEdit extends React.Component {
         };
     }
 
+    componentWillUnmount() {
+        if (!this.state.isFocused) {
+            return;
+        }
+
+        // Show the main composer when the focused message is deleted from another client
```

Comment:
@bernhardoj Could you add an explanation of why we only do it in the focused message and what the problem we trying to solve with that code?

Reply:
updated